explainer.md

```diff
@@ -0,0 +1,369 @@
+method for a user to prove a pre-established identity with a given site.
+However, WebAuthn is not immediately suitable to solve the payments
+authentication problem, as:
+
+1. It requires the ??? to be present in the transaction (e.g. via an embedded
+   iframe), which increases user friction and lowers conversion rates.
+1. The generated assertion contains no payments-related information, which could
+   be useful to meet e.g. [Dynamic Linking] requirements.
+1. WebAuthn does not allow credential creation in a cross-origin iframe, thus
+   excluding a useful onboarding flow - enrolling a Customer after they have
+   completed a traditional authentication flow (e.g. via SMS OTP).
+
+This proposal attempts to provide an authentication solution for online
+payments that is both as strong and low-friction as WebAuthn, whilst also
+solving these three issues.
+
+### Goals
+
+Find a solution that (in no particular order):
+
+* Has as-strong or stronger authentication than current challenge flows.
+* Is less friction for the Customer than current challenge flows.
+* Is more protective of user-privacy than current frictionless flows.
+* Improves the ability for online payments to meet regulations such as [Dynamic
+  Linking].
+
+> **NOTE**: It is out-of-scope for the specification to **prove** adherence
+> to e.g. Dynamic Linking, but we aim to produce a solution that could be
+> vetted as such.
+
+* Allows for in-flow enrollment of Customers during a traditional challenge flow.
+
+### Non-Goals
+
+* Selection of a Payment Instrument by the Customer; it is presumed that the
+  Customer has already done so (e.g. by typing in their credit card number).
+* ID & V to establish real world identity during enrollment; it is up to the
+  ??? to determine the Customer's identity to their satisfaction.
+* Providing authentication for peer-to-peer or business-to-business
+  transactions.
+    * We expect that Secure Payment Confirmation may be useful in these
+      cases, but are concentrating on consumer-to-business cases for now.
+
+## Proposed Solution: Secure Payment Confirmation
+
+Secure Payment Confirmation (SPC) builds on top of [WebAuthn] to add
+payment-specific datastructures to the signed assertion, and to relax
+assumptions to allow the API to be called in payment contexts.
+
+The intention for Secure Payment Confirmation is that a Customer would enroll
+once on a given device for a given account with an ???, either on the ???'s
+website or during a traditionally-authenticated online payment (e.g. after
+completing a challenge in a ??? iframe). Then, in subsequent transactions on
+**any Merchant** that wishes to use Secure Payment Confirmation:
+
+1. The Customer identifies themselves to the Merchant as per normal (e.g.
+   by providing a credit card number)
+1. Using a back-channel (e.g. the 3D-Secure protocol), the Merchant asks for,
+   and receives, a list of credentials for the identified Customer from the ???.
```

Comment:
Nit: Link "credentials" to the WebAuthn concept of credential identifiers (i.e., public key material) for clarity?